azure-cognitiveservices/vision/contentmoderator/pom.xml

```diff
@@ -0,0 +1,91 @@
+<!--
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.microsoft.azure.cognitiveservices</groupId>
+    <artifactId>azure-cognitiveservices-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+    <artifactId>azure-cognitiveservices-contentmoderator</artifactId>
+    <version>1.0.0-beta-SNAPSHOT</version>
```

Comment:
Same here

Reply:
Only Bing related services will go out without the "beta" (except Visual Search). The rest of the libraries will remain beta for now.